src/cli/cli.cpp

```diff
@@ -4793,6 +4723,32 @@ void Interpreter::OutputLine(const char *aFormat, ...)
     OutputFormat("\r\n");
 }
 
+void Interpreter::OutputLine(uint16_t aIndentSize, const char *aFormat, ...)
+{
+    va_list args;
+
+    OutputSpaces(aIndentSize);
+
+    va_start(args, aFormat);
+    OutputFormatV(aFormat, args);
+    va_end(args);
+
+    OutputFormat("\r\n");
+}
+
+void Interpreter::OutputSpaces(uint16_t aCount)
+{
+    static const char kSpaces[] = {' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '};
+    uint16_t          len;
+
+    while (aCount > 0)
+    {
+        len = OT_MIN(aCount, sizeof(kSpaces));
```

Comment:
```suggestion
        uint16_t len = OT_MIN(aCount, sizeof(kSpaces));

```
nit: put `uint16_t len` in smaller scope?